lib/util/market-status-details.ts

```diff
@@ -0,0 +1,78 @@
+import { ZTG } from "lib/constants";
+import Decimal from "decimal.js";
+import type { ScalarRangeType } from "@zeitgeistpm/sdk/dist/types";
+import moment from "moment";
+
+export const getScalarOutcome = (
+  outcome: string,
+  scalarType: ScalarRangeType,
+) => {
+  const dateFormat = "MM.DD.YYYY";
```

Comment:
What's the reason for the specific formatting here? I would rather format based on locale by using `Intl.formatDate`